test/slow/nvm use/Running 'nvm use --save --lts' should work as expected

```diff
@@ -0,0 +1,82 @@
+#!/bin/sh
+
+set -ex
+
+clean () {
+  if [ "$(basename "$(pwd)")" = "tmp" ]; then
+    cd ..
+  fi
+  rm -rf .nvmrc tmp;
```

Comment:
maybe we should store the directory path in a var, create it immediately, and fail if it doesn't work - and then use the var in `clean`? that way we also wouldn't have to rely on someone not running the tests in a folder also called "tmp"

If you make an entire folder for `--save`, then you could do this in `setup` and `teardown` files